website/docs/opengl/c1/2-hello-quad.md

```diff
@@ -0,0 +1,524 @@
+
+The `6` is simply the number of elements in our EBO. Remember, a quad is two triangles, with three vertices per triangle, making six vertices total. 
+
+We tell it we're using an unsigned int as the element type (you may have noticed earlier that `indices` was of type `uint[]`). The most commonly used values are `UnsignedInt` and `UnsignedShort`. Some older GPUs only supported `UnsignedShort`, however all modern GPUs can fully support `UnsignedInt`, so this isn't really something you need to worry about anymore.
+
+The last parameter is a pointer to the starting index of the indices. Since we want all the indices, we just set this value to `null`. (Equivalent to `(void*) 0`).
+
+And that's it! Run your program and you should see a lovely orange rectangle on a blue background. Exciting, isn't it... Right...?
+
+![Final result](../../../images/opengl/chapter1/final-result-t2.png)
+
+While this may have seen like a lot of set up for a boring result, this code can render pretty much anything you want to the screen. It remains pretty much the same, whether you're rendering a basic quad like this, or a complex 3D model. All you need to change are the vertices & indices going in, and some more complex shader code to handle the transformations.
+
+Once you get the basics understood and you can confidently use it, creating more complex stuff becomes a lot easier.
+
+If you would like to view the full source code for the application, you can view it [here](../sources/1.2-final-result.html).
+
+## Cleaning up
+When you're done with an OpenGL resource, you should delete it, to free GPU memory. For this though, it's not needed. The driver will automatically clean all created resources when your application closes. In fact, it's recommended that you *don't* manually delete resources when the application closes. The driver can free these objects a lot faster than manually removing them. If you have a lot of objects, manually deleting can cause the application to hang while the driver tries to free everything.
+
+You **MUST** remember to remove unused objects while your application is running, however, as forgetting to do so will use more and more of the GPU's memory, potentially causing it to run out of memory. OpenGL isn't garbage collected! It requires manual memory management, a lot like C.
+
+## Wrapping up
+That was a lot to digest! You've hopefully learned a lot along the way though, and if there's bits you still don't understand, just go back and re-read the sections again to hopefully improve your understanding. Don't worry about it too much though, the learning curve is steep, and once you do understand it, you'll be writing graphics programs in no time!
+
+Here's some stuff you can do now:
+
+* Move on to the [next tutorial](), where you'll learn how to create a GL context and display a quad on the screen.
```

Comment:
Copy paste typo?

Reply:
Whoops, yes, I didn't even notice it when I read it just now, lol.